python/hail/dataset.py

```diff
@@ -4911,20 +4911,6 @@ def make_table(self, variant_expr, genotype_expr, key=[], separator='.'):
         """Produce a key with one row per variant and one or more columns per sample.
 
         **Examples**
-        
-        >>> kt = vds.make_table('v = v', ['gt = g.gt', 'gq = g.gq'])
-
-        **Notes**
-
-        Per sample field names in the result are formed by
-        concatenating the sample ID with the ``genotype_expr`` left
-        hand side with ``separator``.  If the left hand side is empty::
-
-          `` = expr
-
-        then the dot (.) is omitted.
-
-        **Examples**
 
         Consider a :py:class:`VariantDataset` ``vds`` with 2 variants and 3 samples:
```

Comment:
Will read better if you change `:` to period, since code flows into following sentence.